### PR TITLE
:book: Add the document to adjust the log level 

### PR DESCRIPTION
**Pull request by yanmxa**
## Related issue(s)
https://github.com/stolostron/multicluster-global-hub/issues/1178

https://github.com/stolostron/multicluster-global-hub/pull/1184
https://github.com/stolostron/multicluster-global-hub/pull/1185

Fixes # 

## Tests
* [ ] Unit/function tests have been added and incorporated into `make unit-tests`.
* [ ] Integration tests have been added and incorporated into `make integration-test`.
* [ ] E2E tests have been added and incorporated into `make e2e-test-all`.
* [ ] List other manual tests you have done.

### CODE REVIEW

File: doc/troubleshooting.md [hunk 1]
```diff
@@ -2,6 +2,22 @@
 
 You can run troubleshooting steps to determine issues on your Multicluster Global Hub.
 
+## Change the `mutlicluster-global-hub` Log Level at Runtime
+
+To aid in troubleshooting, you may need detailed log information from the operator, manager, and agent components. Currently, the supported log levels are `debug`, `info`, `warn`, and `error`. For the complete list of available log levels, refer to [zap log levels](https://github.com/uber-go/zap/blob/master/level.go#L30-L49).
```
Comment (clyang82) [inline doc/troubleshooting.md:7]: let us mention what is the default value here?

File: doc/troubleshooting.md [hunk 2]
```diff
@@ -2,6 +2,22 @@
...
+
+To adjust the log level, update the `logLevel` value in `multicluster-global-hub-config` ConfigMap on the global hub cluster. This change will dynamically modify the log level for the operator, manager, and agent without requiring a restart.
```
Comment (clyang82) [inline doc/troubleshooting.md:9]: Do we pass the `logLevel` to agent automatically?

File: doc/troubleshooting.md (hunk 1 above)
Comment (yanmxa) [reply doc/troubleshooting.md]: Done.

**Pull request merged**